client/src/app/api/connect/route.ts

```diff
@@ -0,0 +1,55 @@
+import { Jamsocket } from '@jamsocket/server'
+
+const JAMSOCKET_ACCOUNT = process.env.JAMSOCKET_ACCOUNT
+const JAMSOCKET_SERVICE = process.env.JAMSOCKET_SERVICE
+const JAMSOCKET_TOKEN = process.env.JAMSOCKET_TOKEN
+const JAMSOCKET_DEV = process.env.JAMSOCKET_DEV
+
+let jamsocket: Jamsocket
+if (JAMSOCKET_DEV) {
+  jamsocket = new Jamsocket({
+    dev: true,
+  })
+} else {
+  if (!JAMSOCKET_ACCOUNT || !JAMSOCKET_SERVICE || !JAMSOCKET_TOKEN) {
+    throw new Error(
+      'Missing environment variables JAMSOCKET_ACCOUNT, JAMSOCKET_SERVICE, or JAMSOCKET_TOKEN. ' +
+        'If you intend to run in local dev mode, set JAMSOCKET_DEV=true.',
+    )
+  }
+
+  jamsocket = new Jamsocket({
+    account: JAMSOCKET_ACCOUNT,
+    service: JAMSOCKET_SERVICE,
+    token: JAMSOCKET_TOKEN,
+  })
+}
+
+export async function POST(request: Request) {
+  const { docId } = await request.json()
+  if (typeof docId !== 'string') {
+    return Response.json({ error: 'docId is required in body' }, { status: 400 })
+  }
+
+  // This is a good place to check if the user has permissions
+  // to access the document! If they don't, you should return a 403 error here.
+  // For the sample code, we are assuming that every user can access every document.
+
+  const spawnResult = await jamsocket.connect({
+    key: docId,
+    spawn: {
+      executable: {
+        env: {
+          STORAGE_BUCKET: 'tldraw-jamsocket-demo',
+          STORAGE_PREFIX: docId,
```

Comment:
these are only needed locally, right? IIRC on Jamsocket with S3 integration we set them.

Reply:
yeah, that's right